packages/cryptoplease/lib/features/incoming_split_key_payment/presentation/components/success_content.dart

```diff
@@ -23,8 +24,70 @@ class SuccessContent extends StatelessWidget {
   }
 
   @override
-  Widget build(BuildContext context) => CommonSuccess(
-        text: context.l10n.splitKeySuccessMessage,
-        onClosePressed: () => _onSuccessConfirmed(context),
+  Widget build(BuildContext context) => Scaffold(
```

Comment:
I would extract the whole `StatusScreen` as a separate widget in UI library, with required status and optional content.